code/controllers/communications.dm

```diff
@@ -205,9 +205,13 @@ var/const/RADIO_MAGNETS = "radio_magnet"
 var/const/RADIO_LOGIC = "radio_logic"
 
 var/global/datum/controller/radio/radio_controller
+var/global/list/headsets[0]
 
 /hook/startup/proc/createRadioController()
 	radio_controller = new /datum/controller/radio()
+	for(var/A in headsets)
```

Comment:
Yes but in that case wouldn't all headsets be created *after* the radio controller is made so it doesn't matter?

`createRadioControlle()` is called by https://github.com/ParadiseSS13/Paradise/blob/master/code/world.dm#L49

`initialize()` gets called later down the line through `master_controller.setup()`

Any atom is always `in world`, so the initializations would catch them no matter what.